api/formance.com/v1beta1/authclient_types.go

```diff
@@ -40,14 +41,17 @@ type AuthClientSpec struct {
 	RedirectUris []string `json:"redirectUris,omitempty" yaml:"redirectUris"`
 	// +optional
 	// RedirectUris allow to list allowed post logout redirect uris for the client
-	PostLogoutRedirectUris []string `json:"postLogoutRedirectUris,omitempty" yaml:"PostLogoutRedirectUris"`
+	PostLogoutRedirectUris []string `json:"postLogoutRedirectUris,omitempty" yaml:"postLogoutRedirectUris"`
```

Comment:
Fix Auth config.yaml according to https://github.com/formancehq/auth/blob/bce090421ddee5f1ea35669e7d04a4ee150d2051/pkg/client.go#L154

Reply:
Maybe brankin change ? cc @gfyrag